.github/workflows/integrate.yaml

```diff
@@ -0,0 +1,24 @@
+name: DB Rules & Unit Test Continuous Integration
+
+on:
+  push:
+    branches: [master]
+  pull_request_target:
+    types: [assigned, opened, synchronize, reopened]
```

Comment:
Not needed here, please use normal pull_request

Reply:
Okay